hermes/deletionPicker.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
+//
+// DeletionPicker picks what file from 10 to 50 (files 0-9 are kept in the system for persistence) to delete
+package hermes
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+const (
+	beg                    = 10 // we can delete files staring from the file Hermes_10
+	numberOfDeletableFiles = 41 // there are 41 files to delete from [Hermes_10,Hermes_50]
+)
+
+// PickFileToDelete picks which file to delete and returns a string: file name in the form "Hermes_ID".
```

Comment:
Rather than picking a random number, could we expect an `array` of the files that actually exist, and return one of these? That seems somewhat less fragile than constructing the file name in place. This seems especially relevant given that the filename here doesn't seem to match that generated by `generateFileName`

Reply:
It's the first 9 characters of the name generated by generateFileName then the idea is (If  I understand correctly @evanSpendlove can you confirm?) that the deleteprobe will take that information in and delete the appropriate file whose full name we obtain from the parsed NIL file

Reply:
If we're obtaining the filename from the NIL file, then you should definitely pick the file after you parse the file names from the NIL file., and take an array of files.